features/net/FEATURE_IPV4/TESTS/mbedmicro-net/nist_internet_time_service/main.cpp

```diff
@@ -8,55 +8,64 @@
 #include "greentea-client/test_env.h"
 
 namespace {
-    //const char *HTTP_SERVER_NAME = "utcnist.colorado.edu";
     const char *HTTP_SERVER_NAME = "pool.ntp.org";
     const int HTTP_SERVER_PORT = 123;
```

Comment:
Hi @bridadan 
In my company local setup, boards can not have access to external NTP servers... :-(
But we have some internal NTP server.
So do you think it would be possible to create a new command line option to allow user to set its own NTP sever name ?
Thx